### DISCUSSION TITLE
是否应允许数个小模组在同一PR中提交

### DISCUSSION THREAD

**Discussion by mamaruo** [CFPAOrg/Minecraft-Mod-Language-Package#1770] [Ideas]
起因：@XiaoXin1113 最近提交了数十个仅有几行（很少一部分是几十行）的模组翻译。

是否应该放开小模组多合一的限制，有何利弊？

**Comment by SeaqUs** [comment]
我认为是可以的，毕竟开那么多PR，开的人累，审核的也累

**Comment by n0099** [comment]
以前的全局查找替换不也是一个pr里提交的，比如#1646 
以及为什么用github discussions

**Reply by mamaruo** [reply]
显然这类批量替换类 PR 不能分开。

discussion 的具体用法？

**Reply by n0099** [reply]
discussion都不能连接到pr/issue（pr/issue下面不会有被链接的提示消息）

**Comment by mamaruo** [comment]
提议：当多个模组行数之和小于 100 行时，建议用户合并为一个 PR 提交。

**Reply by n0099** [reply]
![](https://user-images.githubusercontent.com/13030387/163212576-4b6b2999-b272-4ef8-a9da-360d416e49e6.png)

**Reply by mamaruo** [reply]
另外前成员 WuzgXY 的意见是完全支持。 

以及 100 行这个数字需不需要提高一点点。

**Reply by n0099** [reply]
建议200

**Reply by Cyl18** [reply]
其实是个灵活标准 约多少行

**Reply by n0099** [reply]
那就会陷入车轮战

**Reply by Cyl18** [reply]
每个模组都小于60行且所有模组行数加起来小于200行这种？

**Reply by n0099** [reply]
建议bot自动化

**Reply by SeaqUs** [reply]
建议200

**Comment by AnzhiZhang** [comment]
不支持，一旦发生某个模组需要大量讨论，会影响其它正常需要合并或拒绝的模组翻译，应当分开处理。

**Reply by mamaruo** [reply]
某个模组阻止合并这一问题可以通过简单地删除它来解决。

**Reply by n0099** [reply]
感觉不如拆开

**Reply by dovisutu** [reply]
可以把已经通过的部分cherry-pick出来，然后剩下的接着审核吧
